contents/blog/data-warehouse-at-posthog.mdx

```diff
@@ -0,0 +1,353 @@
+---
+title: How we use the data warehouse at PostHog
```

Comment:
Maybe "How we use PostHog's built-in data warehouse" would be better here? I'm probably overthinking this, but I worry the headline implies we're talking about any data warehouse, rather than the one we built.